config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -42,8 +42,8 @@ presets:
     preset-capz-windows-ci-entrypoint-common-main: "true"
     # https://capz.sigs.k8s.io/developers/development.html#running-custom-test-suites-on-capz-clusters
   env:
-    - name: USE_CI_ARTIFACTS
-      value: "true"
+    - name: KUBERNETES_VERSION
+      value: "v1.23.5"
```

Comment:
all of the changes in this test config are actually done in order to make this test work at all; it's currently broken:

https://testgrid.k8s.io/sig-windows-master-release#capz-azuredisk-windows-2019

Reply:
This is job is mean to run against the main kubernetes branch as an integration test. This would pin it to version

Reply:
This env variable is part of the test-infra label `preset-capz-windows-ci-entrypoint-common-main` which is only used in a periodic job that runs a Windows 2019 azuredisk test.

Reply:
The intent of this is job is to run Azure Disk against the `master` branch of kubernetes.  If we don't use `USE_CI_ARTIFACTS` this will use `1.23.5`.  This job was replacing a similiar job we had before. Is there another way we should be doing this?

Reply:
Because this is only running every 48 hours I assume using the "latest" CI build at the time of the test run is O.K.?

E.g.: https://storage.googleapis.com/k8s-release-dev/ci/latest.txt

As of right now that means `v1.25.0-alpha.0.653+fa7309438f7c6d`

Reply:
Done.